### ISSUE TITLE
More PC query support

### ISSUE THREAD

**Issue by ugurdogrusoz** [iVis-at-Bilkent/newt#243]
Should be straightforward to add other type of PC queries (under Query > Pathway Commons). Let's discuss which ones and with what GUI.

**Comment by ugurdogrusoz** [comment]
https://github.com/PathwayCommons/cpath2/issues/296

**Comment by ugurdogrusoz** [comment]
Another PC service issue we revealed was attachment of info boxes (state variables) to OR nodes. Will be fixed with the next PC release.

**Comment by ugurdogrusoz** [comment]
Because state variables belong to EPNs, not logical operators according to SBGN: http://co.mbine.org/specifications/sbgn.pd.level-1.version-1.3.

**Comment by ozgunbabur** [comment]
Yes. Using SBGN OR glyph for a generic physical entity in BioPAX is a hack.
The reason is SBGN still do not support generics. Since OR is not an entity
but a logical operation, it cannot have state variables. Igor, please don't
revert my fix.
